test/integration/api.bats

```diff
@@ -48,3 +48,50 @@ function teardown() {
        [ "${#lines[@]}" -eq  2 ]
        [[ "${lines[1]}" == *"false"* ]]
 }
+
+
+@test "docker create/start/stop/restart/kill should return success" {
+	start_docker 1
+	swarm_manage
+	
+	#create
+	run docker_swarm create --name test_container busybox sleep 1000
```

Comment:
This should be `run docker_swarm create --name test_container busybox sh` to make the container run forever.